pkg/agent/nplagent/lib/utils.go

```diff
@@ -0,0 +1,98 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lib
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+	"k8s.io/klog"
+)
+
+func Stringify(serialize interface{}) string {
```

Comment:
If these funcs are not really used by multiple packages, we could move them to their consumer packages to be together with their consumer code?

I am also trying to see if we can remove this nplagent/lib package.

Reply:
I think we need this to be used is multiple places. If required, can we move this to a place we hold common util functions ?

Reply:
I saw it is called in k8s/ only. If so one way is to move it there.
Or if you think it is generic enough, consider pkg/util/json; or maybe nplagent/util/json.go?

Reply:
Okay. I'll move it to k8s/ for now. If we need to use it in other packages, I'll move it.

Reply:
Sounds good to me.